Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.7.3
+FROM golang:1.7.4
```

Comment:
Consider using `golang:1.7` instead, which I think would just pull the latest go 1.7.x image